dashboard/config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
+# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
+# amazon:
+#   service: S3
+#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+#   region: us-east-1
```

Comment:
Can we grab the region dynamically? If the `CDO` object has been initialized at this point, then we could set it from `CDO.aws_region`

Reply:
We possibly could if we want to, but this is a commented-out section of an auto-generated configuration file which we don't actually use anywhere. It's just a default placeholder which Rails automatically provides for new projects as of v6.0 and which it expects to always be in place as of v6.1, it's not something we need to personalize until we decide to actually start using it.

Reply:
oh, my. I need to improve my reading comprehension. 😂
Yup. these placeholders are fine.